packages/react-drag-drop/src/next/components/DragDrop/DragDropSort.tsx

```diff
@@ -159,7 +163,7 @@ export const DragDropSort: React.FunctionComponent<DragDropSortProps> = ({
             );
         }
       })}
-      <DragOverlay>{activeId && getDragOverlay()}</DragOverlay>
+      {canUseDOM ? ReactDOM.createPortal(dragOverlay, document.getElementById('root')) : dragOverlay}
```

Comment:
I feel like it might be worth adding a prop to be able to customize the element passed to createPortal, though also may be something to wait and see if anyone mentions any issue with it. Not a blocker for me to keep this as-is, so unless anyone else would want to see that update ☑️